functions/memorystore/redis/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2019, Google LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START functions_memorystore_redis]
+
+const redis = require('redis');
+
+const REDISHOST = process.env.REDISHOST || 'localhost';
```

Comment:
Nit: can use use underscores here? i.e. `REDIS_HOST` and `REDIS_PORT`.

Reply:
All the existing samples use `REDISHOST` and `REDISPORT`, so while I agree with you, I think it's better to be consistent, which would also help with writing the docs.